App/src/main/java/cc/softwarefactory/lokki/android/MainApplication.java

```diff
@@ -62,7 +62,9 @@ protected int sizeOf(String key, Bitmap bitmap) {
         if (!iDontWantToSeeString.isEmpty()) {
             try {
                 MainApplication.iDontWantToSee = new JSONObject(iDontWantToSeeString);
-            } catch (JSONException ex) {
+            } catch (JSONException e) {
+                MainApplication.iDontWantToSee = null;
```

Comment:
If I follow the code correctly, in this case it's going to be null anyway – but yeah it's not a bad idea to be explicit.